vuu-ui/showcase/src/examples/Apps/NewTheme.examples.tsx

```diff
@@ -81,7 +74,7 @@ export const ShellWithNewTheme = () => {
         }
         return menuDescriptors;
       },
-      async (action: MenuActionClosePopup) => {
+      (action: MenuActionClosePopup) => {
```

Comment:
a MenuActionClose action has an options object. I made a change to add an attribute to that options object `controlledComponetId`is the id of the component controlled by the tab from which the save layout menu was popped up. That gives you the id of the element we need to capture teh screen grab from. Will avoid that slightly fragile
getElementsByClassName("vuuShell-content")[0] to locate the element.
It can be passed as a prop straight into the SaveLayoutPanel